#!/usr/bin/env python
PACKAGE="path_gen_srv"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cost_scalar",double_t,0,"cost gain",0.7,0,5.0)
gen.add("heuristic_scaler",double_t,0,"heuristic function gain",1,0,10)

exit(gen.generate(PACKAGE,"path_gen_srv","path_srv"))
